src/realtime_metrics/main.py

```diff
@@ -148,6 +180,52 @@ def eta_accuracy(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]]):
     return mean_accuracy
 
 
+def availability_acceptable_stop_time_updates(stop_time_updates: list[tuple[TripUpdate, StopTimeUpdate]], 
+                                              time_frame_start: int, 
+                                              time_frame_end: int):
+    """
+    computes the availability of acceptable stop time updates metrics for the given stop time updates in the given time frame.
+    stop_time_updates: list of corresponding trip updates and stop time updates
+    time_frame_start: start of the time frame to observe, in minutes since 1970
+    time_frame_end: end of the time frame to observe, in minutes since 1970
+    """
```

Comment:
Return value is missing from docs. Might be interesting to know that is is given as percentage